### PR TITLE
refactor: Reorganize imports in `chain/client`

### PR DESCRIPTION
**Pull request by pmnoxx**
Fix grouping dependencies in `chain/client`.

### CODE REVIEW

File: chain/client/src/client_actor.rs [hunk 1]
```diff
@@ -9,8 +9,7 @@ use actix::dev::ToEnvelope;
 use actix::{Actor, Addr, Arbiter, AsyncContext, Context, Handler, Message};
 use actix_rt::ArbiterHandle;
 use borsh::BorshSerialize;
-use chrono::DateTime;
-use chrono::Duration as OldDuration;
+use chrono::{DateTime, Duration as OldDuration};
```
Comment (matklad) [inline chain/client/src/client_actor.rs:12]: Unrelated, but I would avoid an alias, and used `chrono::Duration` instead.

**Review by matklad — APPROVED** [review]

**Review by mzhangmzz — APPROVED** [review]

**Review by bowenwang1996 — DISMISSED** [review]
I suggest we wait until we finalize #5262 and change everything at once to avoid doing it piecemeal and constantly having to fix it

File: chain/client/src/client_actor.rs (hunk 1 above)
Comment (pmnoxx) [reply chain/client/src/client_actor.rs:12]: Fixed, in a different PR.

**Comment by pmnoxx** [comment]
@bowenwang1996 Now that #5262 has been finalize, can we proceed?

**Review by mm-near — APPROVED** [review]

**Comment by bowenwang1996** [comment]
Is there a way to enforce the style? Otherwise what prevents us from breaking it from time to time?

**Comment by pmnoxx** [comment]
> Is there a way to enforce the style? Otherwise what prevents us from breaking it from time to time?

In terms of by only using `cargofmt`: No.

However, the most common reason why the code style becomes inconsistent is because someone.

There are some options though:
- We could write a simple language parses, which checks `*.rs` file if there is an empty line, or a few between two consecutive imports.

**Review by bowenwang1996 — APPROVED** [review]
I'll approve this PR, but I don't think we should spend our effort on fixing every style inconsistencies in the existing codebase, especially because we *know* that it will be broken again. I think we should try to point out style inconsistencies in new PRs and hopefully over time people will get used to what is written in the style guide.

**Review by chefsale — APPROVED** [review]

**Pull request merged**